README.md

```diff
@@ -169,9 +168,9 @@ customElements.get('mr-spacecraft') || customElements.define('mr-spacecraft', Sp
 
 #### Systems
 
-A System contains logic that is applied to all entities that have a corresponding Component, using the data stored by the component. Unlike Entities & Components, Systems have no HTML representation and are implemented entirely in JS.
+A System contains logic that is applied to all entities that have a corresponding Component, using the data stored by the component. Unlike Entities & Components, Systems have no HTML representation and are implemented entirely in JavaScript.
 
-When a component is attached or detatched from an entity, it is added or removed from its System's registry of entities.
+When a component is attached or detached from an entity, it is added or removed from its System's registry of entities.
```

Comment:
need to add a to after attached, so - `attached to` , since we have detached from